pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -132,6 +132,9 @@ protected void testQuery(String pinotQuery, String h2Query)
   @AfterClass
   public void tearDown()
       throws Exception {
+    // Setting data table version back
+    DataTableBuilderFactory.setDataTableVersion(DataTableBuilderFactory.DEFAULT_VERSION);
```

Comment:
Do we need to set the following to the `setUp` method of this integration test class as well?
```
    // Setting data table version to 4
    DataTableBuilderFactory.setDataTableVersion(DataTableFactory.VERSION_4);
```

Reply:
yeah this is a hotfix, multistage test was already setting it in setup but it wasn't set back in teardown i just happen to notice